fpga.chart.py

```diff
@@ -62,7 +62,12 @@ def __init__(self, configuration=None, name=None):
         self.xilinx_cmd = self.configuration.get('xilinx_cmd')
         self.fpga_count = 1
         self.dsn = self.configuration.get('dsn')
-        self.metrics = [ 'bytes', 'jobs', 'max', 'temps', 'powers' ]
+        self.check_temp_power = self.configuration.get('check_temp_power')
+
+        if self.check_temp_power is True:
```

Comment:
Proposal to reduce code:

    self.metrics = [ 'bytes', 'jobs', 'max' ]
    if self.check_temp_power:
        self.metrics.extend(['temps', 'powers'])